charts/refinery/templates/deployment-redis.yaml

```diff
@@ -16,7 +16,8 @@ spec:
   {{- include "refinery.redis.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
+      {{- $annotationsMerged := merge .Values.redis.podAnnotations .Values.podAnnotations }}
+      {{- with $annotationsMerged }}
```

Comment:
Sorry reviewing this took so long.  Can you move this logic into a helper template?